pkg/virtctl/guestfs/guestfs.go

```diff
@@ -78,7 +78,7 @@ func NewGuestfsShellCommand(clientConfig clientcmd.ClientConfig) *cobra.Command
 	cmd.PersistentFlags().BoolVar(&kvm, "kvm", true, "Use kvm for the libguestfs-tools container")
 	cmd.PersistentFlags().BoolVar(&root, "root", false, "Set uid 0 for the libguestfs-tool container")
 	cmd.SetUsageTemplate(templates.UsageTemplate())
-	cmd.PersistentFlags().StringVar(&group, "group", "", "Set the gid and fsgroup for the libguestfs-tool container")
+	cmd.PersistentFlags().StringVar(&fsGroup, "fsGroup", "", "Set the fsgroup for the libguestfs-tool container")
```

Comment:
This seems to be a breaking change, why don't we just add the fsgroup option?

Reply:
The `group` flag was used to specify both `fsGroup` and `runAsGroup` for the pod manifest. The problem is that having `runAsGroup` without `runAsUser` violates the spec (ref. in the PR description). So this is not about replacing `group` with `fsGroup` but rather allowing *only* `fsGroup`.

Reply:
But we do set `runASUser` in some cases and I believe we need to specify the group on some platforms. I am ok with the split(group for group, fsgroup for fsgroup)

Reply:
Currently, we do not set `runAsUser` for guestfs pod manifest. I think @alicefr mentioned that there is a PR for that https://github.com/kubevirt/kubevirt/pull/8336. The problem is that if we set only `runAsGroup`, the pod creation fails. So this is a bug. To fix it, I think there are two approaches:
- disallow setting `runAsGroup` (that is what this PR does)
- allow setting `runAsGroup` **only** along with `runAsUser` (probably that is what Alice plans to do in that PR... tbh, I haven't looked at it yet).

Reply:
@vasiliy-ul @xpivarc the PR https://github.com/kubevirt/kubevirt/pull/8336 (temporarily incudes Vasily's change) and adds on top the `--gid` flag. Lubo, we haven't merged the support for `runAsUser` yet

Reply:
Hm... https://github.com/kubevirt/kubevirt/pull/8336 seems to be ready to get in... maybe then let's just close this PR in favor of that one?

Reply:
@vasiliy-ul you prefer. It contains your commit of this PR

Reply:
Definitely, let's unhold https://github.com/kubevirt/kubevirt/pull/8336. I will then close this one if no objections.